README.md

```diff
@@ -124,3 +124,36 @@ TXT record, usually labeled "Seed API" to your HTTP API endpoint, eg.
 These records can be set on the web client at <https://app.radicle.network/registrations/acme>
 for an org registered under `acme.radicle.eth`. Replace `acme` with the name
 of your org.
+
+### Docker
+
+There are `Dockerfile` provided for both services in the respective directories.
+
+#### Building
+
+To build the containers, run:
+
+    $ docker build -t radicle-services/org-node -f org-node/Dockerfile .
+    $ docker build -t radicle-services/http-api -f http-api/Dockerfile .
+
+#### Running
+
+To run the org node after it is built, you can run for example:
+
+    $ docker run radicle-services/org-node \
+        -v $HOME/.radicle:/app/radicle -- \
+        --subgraph https://api.thegraph.com/subgraphs/name/radicle-dev/radicle-orgs \
+        --orgs 0xceAa01bd5A428d2910C82BBEfE1Bc7a8Cc6207D9
```

Comment:
The `-v` needs to go *before* the image name, so:

```
docker run -v $HOME/.radicle:/app/radicle \
        radicle-services/org-node \
        --subgraph https://api.thegraph.com/subgraphs/name/radicle-dev/radicle-orgs \
        --orgs 0xceAa01bd5A428d2910C82BBEfE1Bc7a8Cc6207D9
```

Reply:
Aha!